### PR TITLE
Added .env.example

### PR DESCRIPTION
**Pull request by CheatCodeSam**
It could be confusing for new developers to get set up without knowing which environmental variables to set up beforehand. I created a .env.example to help them out.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -0,0 +1,2 @@
+SUBLINKS_API_BASE_URL=
+NODE_ENV=
```
Comment (kgilles) [inline .env.example:2]: `NODE_ENV` is not expected to be defined by the developer. At least not unless they're troubleshooting a non-local environment.

I think we can skip it here. That way it won't appear to be a value devs have to set themselves.

**Review by kgilles — CHANGES_REQUESTED** [review]
Good idea!

Would you mind adding a short blurb about this env var(what it's for) to the env file, and mention the file in the readme?

**Comment by CheatCodeSam** [comment]
Just made those updates. Let me know what you think.

**Review by kgilles — APPROVED** [review]

**Comment by kgilles** [comment]
> Just made those updates. Let me know what you think.

Looks good, thanks!

**Pull request merged**